SPECTRecon.json

```
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Slicer/Slicer/main/Schemas/slicer-extension-catalog-entry-schema-v1.0.0.json#",
+  "build_dependencies": [],
+  "build_subdirectory": ".",
+  "category": "Tomographic Reconstruction",
+  "scm_revision": "main",
+  "scm_url": "https://github.com/PyTomography/Slicer.git"

```

To match https://github.com/PyTomography/SlicerSPECTRecon
```suggestion
  "scm_url": "https://github.com/PyTomography/SlicerSPECTRecon.git"

```

---

Should be fixed now!

---

@lukepolson I do not see a change to the commit in this PR to update the JSON file to match the above code change.

---

@jamesobutler https://github.com/Slicer/ExtensionsIndex/pull/2083/commits/f0c23013fcc246ac545110db268a4c6b4c8efb9e

---

@jamesobutler apologies, it was changed to the wrong name. Now it should be fixed.